RecoLocalMuon/CSCSegment/src/CSCSegAlgoRU.h

```diff
@@ -162,7 +162,7 @@ class CSCSegAlgoRU : public CSCSegmentAlgorithm {
     float wideSeg;
     int minLayersApart;
     bool debugInfo;
-
+    bool enlarge = false;
```

Comment:
this should be a configurable parameter.
Please propagate an appropriate logic to set it via the CSCSegmentProducer configuration

Reply:
Hi Slava!

Do you mean to add it as a parameter to the python config file particularly to RecoLocalMuon/CSCSegment/python/CSCSegmentAlgorithmRU_cfi.py ?

Reply:
right.
The goal is to allow this flag to be changed at configuration step so that some special relval production can be made.

Reply:
Ok. I am on it)